.gitignore

```diff
@@ -0,0 +1,133 @@
+# Created by https://www.gitignore.io/api/xcode,macos,swift
+# Edit at https://www.gitignore.io/?templates=xcode,macos,swift
```

Comment:
It looks like you didn't include Objective-C; I'm not sure if that actually adds anything not included here but thought it worth mentioning!